src/builders.ts

```diff
@@ -428,9 +429,24 @@ export class BomBuilder {
 
     // older npm-ls versions (v6) hide properties behind a `_`
     const isDev = (data.dev ?? data._development) === true
-    if (isDev && this.omitDependencyTypes.has('dev')) {
-      this.console.debug('DEBUG | omit dev component: %j %j', data.name, data._id)
-      return false
+    // if (isDev && this.omitDependencyTypes.has('dev')) {
+    //   this.console.debug('DEBUG | omit dev component: %j %j', data.name, data._id)
+    //   return false
+    // }
+
+    // Initialize component with a default value
+    let component: Models.Component | undefined = undefined;
```

Comment:
this new variable, it is assigned but never used.
so it is useless all along

Reply:
I have to assign the component variable a default value as undefined and then can mark it excluded in necessary place

Reply:
did you generate the test results and check if your changes have any effect?